docs/getting-started/configure-resource-collection/index.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_label: "2. Configure Resource Collection"
+sidebar_position: 2
+pagination_prev: getting-started/install-resoto/index
+---
+
+# Configure Resource Collection
```

Comment:
I would for the term `Cloud Provider` -  I fear Resource Collection is to generic to get understood.

Reply:
See my comment in the PR description; I worry about when we add support for deploying Resoto to various cloud providers.

What about "Configure Cloud Resource Collection"? Or "Connect Cloud Providers"? 🤔

Reply:
"Configuring Cloud Providers" isn't super accurate, because you aren't really configuring the providers.  Rather, you're configuring Resoto to authenticate with providers...

Reply:
@aquamatthias I changed it from "Configure Resource Collection" to "Configure Cloud Resource Collection." Thoughts?  Again, I'm worried about being vague with the verbiage here; we aren't configuring the cloud providers, but configuring Resoto to collect from the providers...

Reply:
If I want to configure access to AWS I do this with [`aws config`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html), configuring access to kubernetes happens though [kubernetes config](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). I don't think anybody will misunderstand...
Strong objections against `Cloud Resource Collection` - this will confuse everybody.
Maybe we can use the term: `Cloud Provider Access`.

Reply:
`Cloud Provider Access` is much better! Will update.